web-ui/src/main/java/org/archcnl/ui/input/ruleeditor/ArchitectureRulesLayout.java

```diff
@@ -49,6 +52,10 @@ private void updateRules() {
         }
     }
 
+    public void switchToNewArchitectureRuleView() {
```

Comment:
I would prefer having the rule creation view integrated into the normal `ArchitectureRulesLayout` so the user can see the already existing rules while creating a new one. But this implementation is fine for now.